Vagrantfile

```diff
@@ -17,8 +19,8 @@ die() {
 echo "Installing build artifacts on the host"
 
 mkdir -p /srv/scality || die "Failed to create /srv/scality"
-rm -f /srv/scality/metalk8s-dev || die "Failed to unlink symlink destination"
-ln -s /vagrant/_build/root /srv/scality/metalk8s-dev || die "Failed to create symlink"
+rm -f /srv/scality/metalk8s-${SHORT_VERSION} || die "Failed to unlink symlink destination"
```

Comment:
Not really important, but here the curly braces are useless and your var should be protected with double quotes to avoid problem while expanding the variables with white characters (even if there are no whitespaces in this variable, this is a good practice).

Reply:
Done